File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_cpp/p02898/p02898_num5_parsed.cpp
#include <iostream>
#include <vector>

using namespace std;

int main() {
    int N, K;
    cin >> N >> K;

    int count = 0;
    vector<int> heights(N);

    for (int i = 0; i < N; i++) {
        cin >> heights[i];
        if (heights[i] >= K) {
            count++;
        }
    }

    cout << count;

    return 0;
}

File: Datasets/TrickyBugs/GenProgs/dpp_generated_progs_python/p02898/p02898_num5_parsed.py
def ride_roller_coaster(N, K, heights):
    count = 0
    for height in heights:
        if height >= K:
            count += 1
    return count

N, K = map(int, input().split())
heights = list(map(int, input().split()))

result = ride_roller_coaster(N, K, heights)
print(result)